package.json

```diff
@@ -26,26 +26,26 @@
   "author": "Samuel Fortier-Galarneau",
   "dependencies": {
     "backoff-func": "^0.1.2",
-    "bluebird": "^2.9.34",
+    "bluebird": "^3.5.2",
+    "lodash": "^4.17.10",
     "request": "^2.34.0",
     "swagger-client": "2.0.26",
-    "underscore": "^1.6.0",
     "uuid": "^3.0.0",
-    "ws": "^1.1.1"
+    "ws": "^6.0.0"
```

Comment:
fwiw, there've been breaking changes between 1.1.1 and 6.0.0

Reply:
I went through the breaking changes. The main one is that this will have to be merged after we move to node 8 as node 4 is no longer supported. I could not find anything else that affects us and did testing with Asterisk locally to try to identify any issues and was unable to detect any issues.

Reply:
I guess it's a moot point if we're planning on doing a major version bump to move from node 4 -> node 8 anyway.